flink-connector-redis/src/main/java/org/apache/flink/streaming/connectors/redis/RedisSink.java

```diff
@@ -144,6 +155,9 @@ public void invoke(IN input, Context context) throws Exception {
             case SET:
                 this.redisCommandsContainer.set(key, value);
                 break;
+            case SETEX:
+                this.redisCommandsContainer.setex(key, this.additionalTTL, value);
```

Comment:
Why all commands that uses `SETEX` must have the same TTL? Why can't it be variable as `HSET`?